### PR TITLE
Feature/overviewfixes

### PR DESCRIPTION
**Pull request by GlenHani**

### CODE REVIEW

File: INZFS.MVC/Views/FundApplication/FinanceProposalOverview.cshtml [hunk 1]
```diff
@@ -40,24 +40,74 @@
 
                     <div class="govuk">
 
-                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">Questions<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
+                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">nbsp&<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
```
Comment (sunilbeis) [inline INZFS.MVC/Views/FundApplication/FinanceProposalOverview.cshtml:43]: This should be `&nbsp;`

File: INZFS.MVC/Views/FundApplication/ProposalSummaryOverview.cshtml [hunk 2]
```diff
@@ -36,13 +36,10 @@
...
 
                 <div class="govuk-tabs" data-module="govuk-tabs">
-                    <h2 class="govuk-tabs__title">
-                        4. Proposal summary
-                    </h2>
 
                     <div class="govuk">
 
-                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">4. Proposal summary<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
+                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">&nbsp<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
```
Comment (sunilbeis) [inline INZFS.MVC/Views/FundApplication/ProposalSummaryOverview.cshtml:42]: This should be `&nbsp;`

File: INZFS.MVC/Views/FundApplication/SummaryOfFinances.cshtml [hunk 3]
```diff
@@ -26,21 +26,18 @@
...
 
                 <div class="govuk-tabs" data-module="govuk-tabs">
-                    <h2 class="govuk-tabs__title">
-                        2. Summary of finances
-                    </h2>
 
                     <div class="govuk">
 
-                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">2. Summary of finances<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
+                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">&nbsp<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
```
Comment (sunilbeis) [inline INZFS.MVC/Views/FundApplication/SummaryOfFinances.cshtml:40]: This should be `&nbsp;`

File: INZFS.MVC/Views/FundApplication/YourDetailsAndEligibility.cshtml [hunk 4]
```diff
@@ -26,21 +26,18 @@
...
                 <div class="govuk-tabs" data-module="govuk-tabs">
-                    <h2 class="govuk-tabs__title">
-                        1. Your details and eligibility
-                    </h2>
-
+                   
                     <div class="govuk">
 
-                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">1. Your details and eligibility<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
+                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">&nbsp<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
```
Comment (sunilbeis) [inline INZFS.MVC/Views/FundApplication/YourDetailsAndEligibility.cshtml:40]: This should be `&nbsp;`

File: INZFS.MVC/Views/FundApplication/YourOrganisation.cshtml [hunk 5]
```diff
@@ -36,13 +36,10 @@
...
                 <div class="govuk-tabs" data-module="govuk-tabs">
-                    <h2 class="govuk-tabs__title">
-                        3. Your Organisation (and partners)
-                    </h2>
-
                     <div class="govuk">
 
-                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">About your organisation
+                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">
+                            &nbsp
```
Comment (sunilbeis) [inline INZFS.MVC/Views/FundApplication/YourOrganisation.cshtml:42]: This should be `&nbsp;`

**Review by sunilbeis — CHANGES_REQUESTED** [review]
Please fix minor issues

File: INZFS.MVC/Views/FundApplication/YourOrganisation.cshtml (hunk 5 above)
Comment (GlenHani) [reply INZFS.MVC/Views/FundApplication/YourOrganisation.cshtml]: done

File: INZFS.MVC/Views/FundApplication/YourDetailsAndEligibility.cshtml (hunk 4 above)
Comment (GlenHani) [reply INZFS.MVC/Views/FundApplication/YourDetailsAndEligibility.cshtml]: done

File: INZFS.MVC/Views/FundApplication/SummaryOfFinances.cshtml (hunk 3 above)
Comment (GlenHani) [reply INZFS.MVC/Views/FundApplication/SummaryOfFinances.cshtml]: done

File: INZFS.MVC/Views/FundApplication/ProposalSummaryOverview.cshtml (hunk 2 above)
Comment (GlenHani) [reply INZFS.MVC/Views/FundApplication/ProposalSummaryOverview.cshtml]: done

File: INZFS.MVC/Views/FundApplication/FinanceProposalOverview.cshtml (hunk 1 above)
Comment (GlenHani) [reply INZFS.MVC/Views/FundApplication/FinanceProposalOverview.cshtml]: done

File: INZFS.MVC/Views/FundApplication/FinanceProposalOverview.cshtml [hunk 6]
```diff
@@ -40,24 +40,74 @@
 
                     <div class="govuk">
 
-                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">Questions<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
+                        <p class="govuk-body-l govuk-!-font-weight-bold govuk-!-margin-bottom-0">&nbsp;&<span class="summary-list-section summary-list-section--right"> @Model.TotalQuestionsCompleted out of  @Model.TotalQuestions completed</span></p>
```
Comment (sunilbeis) [inline INZFS.MVC/Views/FundApplication/FinanceProposalOverview.cshtml:43]: This should be `&nbsp;`

**Review by sunilbeis — CHANGES_REQUESTED** [review]

File: INZFS.MVC/Views/FundApplication/FinanceProposalOverview.cshtml (hunk 6 above)
Comment (GlenHani) [reply INZFS.MVC/Views/FundApplication/FinanceProposalOverview.cshtml]: DONE

**Review by sunilbeis — APPROVED** [review]

**Review by BEISFathir — APPROVED** [review]
Approved

**Review by RSA99 — APPROVED** [review]
Reviewed & Approved

**Pull request merged**